File: Leetcode/1672/1672.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int m = (int)accounts.size();
        int n = accounts.front().size();
        vector<int> clients(m);
        for (int i = 0; i < m; ++i) {
            for (int j = 0; j < n; ++j) {
                clients.at(i) += accounts.at(i).at(j);
            }
        }
        return *max_element(clients.begin(), clients.end());
    }
};

File: Leetcode/1672/1672.go
func max(a, b int) int {
	if a > b {
		return a
	}
	return b
}

func maximumWealth(accounts [][]int) int {
	ans := -1
	for i := 0; i < len(accounts); i++ {
		sum := 0
		for j := 0; j < len(accounts[i]); j++ {
			sum += accounts[i][j]
		}
		ans = max(ans, sum)
	}
	return ans
}
